pkg/kv/import.go

```diff
@@ -127,16 +103,35 @@ func Import(ctx context.Context, reader io.Reader, store Store) error {
 	})
 	log.Info("Processing file")
 
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
```

Comment:
Why do we need this?

Reply:
prevent resource leak

Reply:
https://pkg.go.dev/context#WithCancel
with the example as good resource